clusterloader2/testing/density/config.yaml

```diff
@@ -37,16 +53,14 @@ steps:
     - basename: saturation-rc
       objectTemplatePath: rc.yaml
       templateFillMap:
-        Replicas: {{$replicas}}
+        Replicas: {{$PODS_PER_NAMESPACE}}
+        RCKind: saturationRC
```

Comment:
I think I wasn't clear enough. I don't want to have two separate labels - this should be the same label.
It's very common pattern that you propagate a bunch of labels of grouping object to the pod template.

We should reuse the same label (let's call it "group") and set it to "saturation" or "latency".